src/BMPEncoder.ts

```diff
@@ -35,6 +35,18 @@ export interface ImageCodec {
    * Vertical number of pixels per meter.
    */
   yPixelsPerMeter?: number;
+  /**
+   * Image compression type.
+   */
+  compression: number;
+  /**
+   * Defines which bits represent which color.
+   */
+  colorMasks: number[];
+  /**
+   * Defines how colors are represented.
+   */
+  logicalColorSpace: number;
```

Comment:
Is there a link somewhere in the repo to the specification you used which define those headers?

If not can you add it? (Maybe in the readme). And if those headers have an identifier in the spec you should add them here for context.